lib/rules/yoda.js

```diff
@@ -89,7 +87,7 @@ function looksLikeLiteral(node) {
  *  4. The Literal node which has the `defaultValue` argument if it exists.
  *  5. Otherwise `null`.
  */
-function getNormalizedLiteral(node, defaultValue) {
+function getNormalizedLiteral(node) {
```

Comment:
> 4. The Literal node which has the `defaultValue` argument if it exists.

Could you please edit the comments? I think it should be removed.

Reply:
thanks for pointing . 
Fixed 👍